src/Octoshift/GithubApi.cs

```diff
@@ -360,7 +359,23 @@ public virtual async Task<string> StartOrganizationMigration(string sourceOrgUrl
 
             return (string)data["data"]["startOrganizationMigration"]["orgMigration"]["id"];
         }
-        
+
+        public virtual async Task<string> GetOrganizationMigrationState(string migrationId)
+        {
+            var url = $"{_apiUrl}/graphql";
+
+            var query = "query($id: ID!)";
+            var gql = "node(id: $id) { ... on OrganizationMigration { state } }";
+
+            var payload = new { query = $"{query} {{ {gql} }}", variables = new { id = migrationId } };
+
+            var response = await _retryPolicy.HttpRetry(async () => await _client.PostAsync(url, payload),
+                ex => ex.StatusCode == HttpStatusCode.BadGateway);
```

Comment:
```suggestion
                _ => true);
```
we're getting more aggressive with our retry logic lately, I would change this to retry on any Http error.